crates/cli/src/subcommands/upgrade.rs

```diff
@@ -152,10 +153,8 @@ pub async fn exec(_config: Config, args: &ArgMatches) -> Result<(), anyhow::Erro
 
     if release_version == version::CLI_VERSION {
         println!("You're already running the latest version: {}", version::CLI_VERSION);
-        if !force {
+        if !confirm_prompt(force, "Do you want to reinstall? ")? {
             return Ok(());
-        } else {
-            println!("Force flag is set, continuing with upgrade.");
```

Comment:
Here we added a **new** confirmation prompt, to bring the behavior in line with all the other commands that have a `--force`.